django_cloud_deploy/cli/prompt.py

```diff
@@ -714,6 +714,11 @@ def prompt(self, console: io.IO, step: str,
             creds = self.auth_client.create_default_credentials()
         else:
             creds = self.auth_client.get_default_credentials()
+            if not creds:
+                console.ask(
+                    ('Warning: Your existing credentials file is broken. Press '
```

Comment:
Broken doesnt explain to me what went wrong, anyway we can be more expressive?  is there a way to give them a link with more information?

Reply:
I was not able to find the documentation for the correct credentials json file. So I just made the warning message more specific.